README.adoc

```diff
@@ -1,4 +1,4 @@
-= C++ example project scanned on SonarCloud using GitHub Actions
+= C++ example project scanned on SonarCloud with code coverage using GitHub Actions
 // URIs:
 :uri-qg-status: https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_windows-msbuild-gh-actions-sc
```

Comment:
These 4 links and the link in the bold below (SonarCloud) are outdated

Reply:
Oups, yes, good catch!